test/langtools/tools/doclint/html/HtmlVersionTagsAttrsTest.java

```diff
@@ -1,14 +1,14 @@
 /*
  * @test /nodynamiccopyright/
- * @bug 8072945
+ * @bug 8072945 8247957
  * @summary test tags and attributes specific to the output HTML version
  * @library ..
  * @modules jdk.javadoc/jdk.javadoc.internal.doclint
  * @build DocLintTester
- * @run main DocLintTester -XhtmlVersion:html5 -ref HtmlVersionTagsAttrsTestHtml5.out HtmlVersionTagsAttrsTest.java
- * @run main DocLintTester -XhtmlVersion:html4 -ref HtmlVersionTagsAttrsTestHtml4.out HtmlVersionTagsAttrsTest.java
+ * @run main DocLintTester -XhtmlVersion:html5 -Xmaxerrs 200 -ref HtmlVersionTagsAttrsTest.out HtmlVersionTagsAttrsTest.java
  * @run main DocLintTester -badargs -XhtmlVersion: HtmlVersionTagsAttrsTest.java
- * @run main DocLintTester -ref HtmlVersionTagsAttrsTestHtml4.out HtmlVersionTagsAttrsTest.java
+ *
+ *
```

Comment:
I'm guessing these blank lines are to maintain line numbers, right?

Reply:
Right